src/dotnet/MLSearch.Service/Module/MLSearchServiceModule.cs

```diff
@@ -77,6 +76,7 @@ protected override void InjectServices(IServiceCollection services)
                 services.AddSingleton(static c => c.CreateInstance<ChatIndexInitializer>(ShardScheme.MLSearchBackend))
                     .AddAlias<IChatIndexInitializer, ChatIndexInitializer>()
                     .AddAlias<IHostedService, ChatIndexInitializer>();
+                rpcHost.AddBackend<IChatIndexTrigger, ChatIndexTrigger>();
```

Comment:
It looks ok. But I'd consider keeping this registration near other `rpcHost` registrations with its own `if` condition.
```
// ...
rpcHost.AddBackend<IMLSearchBackend, MLSearchBackend>()
if (!isBackendClient) {
    rpcHost.AddBackend<IChatIndexTrigger, ChatIndexTrigger>();
}
```
What do you think?